jwst/cube_build/ifu_cube.py

```diff
@@ -1367,7 +1367,8 @@ def setup_ifucube(self, j):
         ifucube_model.meta.ifu.dq_extension = 'DQ'
         ifucube_model.meta.ifu.roi_spatial = self.rois
         ifucube_model.meta.ifu.roi_wave = self.roiw
```

Comment:
Since `rois` and `roiw` are both parameters that can be set from the `strun` command line, they also will need to be coerced to a string, i.e.:
```suggestion
        ifucube_model.meta.ifu.roi_spatial = str(self.rois)
        ifucube_model.meta.ifu.roi_wave = str(self.roiw)
```